neonvm/runner/main.go

```diff
@@ -558,6 +583,22 @@ func runInitScript(logger *zap.Logger, script string) error {
 	return nil
 }
 
+func qemuDiskArgs(id string, file string, extraOptions string, deviceExtraOptions string, diskIdxCnt *int) []string {
+	if extraOptions != "" && extraOptions[0] != ',' {
+		extraOptions = "," + extraOptions
+	}
+	if deviceExtraOptions != "" && deviceExtraOptions[0] != ',' {
+		deviceExtraOptions = "," + deviceExtraOptions
+	}
+
+	args := []string{}
+	args = append(args, "-drive", fmt.Sprintf("id=%s,file=%s,if=none,index=%d%s", id, file, *diskIdxCnt, extraOptions))
+	args = append(args, "-device", fmt.Sprintf("virtio-blk,drive=%s%s", id, deviceExtraOptions))
```

Comment:
Why are we adding a device here? I thought it might be necessary only for block devices but here we add it for all the drives. Previously we only had `-drive` for disks.

Reply:
if we don't do it for all devices, it does not boot unfortunately 🤣